agent/engine/dockerclient/versionsupport_windows.go

```diff
@@ -46,9 +46,18 @@ func getWindowsReplaceableVersions() []DockerVersion {
 	}
 }
 
-// getAgentVersions for Windows should return all of the replaceable versions plus additional versions
+// getWindowsSupportedVersions returns the set of remote api versions that are
+// supported by agent in windows
+func getWindowsSupportedVersions() []DockerVersion {
+	return []DockerVersion{
+		Version_1_24,
+		Version_1_30,
+	}
+}
+
+// getAgentVersions for Windows should return all of the replaceable versions plus supported versions
 func getAgentVersions() []DockerVersion {
-	return append(getWindowsReplaceableVersions(), minDockerAPIVersion)
+	return append(getWindowsReplaceableVersions(), getWindowsSupportedVersions()...)
 }
 
 // getDefaultVersion returns agent's default version of the Docker API
```

Comment:
do we keep the default version as `minDockerAPIVersion`?

Reply:
yes, since we know that should at least be the minimum version supported in the instance.